### PR TITLE
BACKLOG-15789: Add page permission check on edit/export in jcontent

### PR DESCRIPTION
**Pull request by gflores-jahia**
## JIRA

https://jira.jahia.org/browse/BACKLOG-15789

## Description

Add separate actions for edit page/content; add permission check on edit page action

## Tests

The following are included in this PR
- [ ] Unit Tests (Most changes _should_ have unit tests)
- [ ] Integration Tests

## Checklist

I have considered the following implications of my change: 

- [ ] Security (in particular for changes to authentication, authorization, data fetching, ...)
- [ ] Performance
- [ ] Migration
- [ ] Code maintainability

## Documentation

- [ ] Inline documentation
- [ ] Internal Documentation (wiki)
- [ ] User-facing Documentation

### CODE REVIEW

File: src/javascript/Edit/Edit.actions.js [hunk 1]
```diff
@@ -9,12 +9,22 @@ import OpenWorkInProgressModalAction from '~/EditPanel/WorkInProgress/OpenWorkIn
...
 
 export const registerEditActions = actionsRegistry => {
-    // Edit action button in JContent
+    // Edit action button in JContent; need separate actions for content and pages
     actionsRegistry.add('action', 'edit', editContentAction, {
         buttonIcon: <Edit/>,
         buttonLabel: 'content-editor:label.contentEditor.edit.contentEdit',
         targets: ['contentActions:2'],
-        hideOnNodeTypes: ['jnt:virtualsite'],
+        hideOnNodeTypes: ['jnt:virtualsite', 'jnt:page'], // For edit content
```
Comment (XiangYuHe) [inline src/javascript/Edit/Edit.actions.js:17]: Just curious why is this added

File: src/javascript/Edit/Edit.actions.js [hunk 2]
```diff
@@ -9,12 +9,22 @@ import OpenWorkInProgressModalAction from '~/EditPanel/WorkInProgress/OpenWorkIn
...
+        getDisplayName: true
+    });
+
+    // Edit action button in JContent; need separate actions for content and pages
+    actionsRegistry.add('action', 'editPage', editContentAction, {
```
Comment (XiangYuHe) [inline src/javascript/Edit/Edit.actions.js:22]: Would greatly appreciated to know why we are separating the actions here

Comment (gflores-jahia) [reply src/javascript/Edit/Edit.actions.js:22]: need separate permissions based off node types

File: src/javascript/Edit/Edit.actions.js (hunk 1 above)
Comment (gflores-jahia) [reply src/javascript/Edit/Edit.actions.js:17]: jnt:page is already included in editPage action. need to hide here so we don't get duplicates.

**Review by tdraier — APPROVED** [review]
